nginx.conf

```diff
@@ -1,104 +1,207 @@
-#Must be set in the global scope see: https://forum.nginx.org/read.php?2,152294,152294
-#Why this is important especially with Plex as it makes a lot of requests http://vincent.bernat.im/en/blog/2011-ssl-session-reuse-rfc5077.html / https://www.peterbe.com/plog/ssl_session_cache-ab
-ssl_session_cache shared:SSL:10m;
-ssl_session_timeout 10m;
+sendfile on;
+tcp_nopush on;
+tcp_nodelay on;
+keepalive_timeout 65;
+types_hash_max_size 2048;
+server_names_hash_bucket_size 512;
+server_names_hash_max_size 512;
+proxy_headers_hash_bucket_size 512;
+proxy_headers_hash_max_size 512;
+include /etc/nginx/mime.types;
+default_type application/octet-stream;
+
+# SSL Config from https://ssl-config.mozilla.org/#server=nginx&version=1.17.7&config=intermediate&openssl=1.1.1k&guideline=5.6
+ssl_session_cache shared:MozSSL:10m;
+ssl_session_timeout 1d;
 
 #Upstream to Plex
 upstream plex_backend {
-	#Set this to the IP address that appears in `ifconfig` (NATTED LAN IP or Public IP address) if you want the bandwidth meter in the server status page to work
```

Comment:
Why have you removed this comment? The bandwidth charts in dashboard (Plex Pass feature) no longer work when using your exact config. Changing it to point to my local LAN IP (192.168.1.222), the bandwidth charts work again perfectly. (And yes, I'm using the set_real_ip_from and real_ip_header and such correctly)